JS-Algorithms/challenges.js

```diff
@@ -42,6 +46,9 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+  if(index >= COUNTRY_NAMES.length) index = index % COUNTRY_NAMES.length;
+  let new_arr = COUNTRY_NAMES.slice(index).concat(COUNTRY_NAMES.slice(0, index));
+  return new_arr;
```

Comment:
You could return the value directly `return COUNTRY_NAMES.slice(index).concat(COUNTRY_NAMES.slice(0, index));`